### ISSUE TITLE
Module '"ngx-sortablejs"' has no exported member 'SortablejsOptions'.ts in Angular 12

### ISSUE THREAD

**Issue by kibria06cse** [SortableJS/angular-sortablejs#251]
I am using Angular 12 version and ngx-sortablejs v11.1.0. 

I used this options on a previous angular version. But whenever I upgrade from the angular and ngx-sortablejs version, I am getting this error.

Is there any alternatitve?

**Comment by tehZeno** [comment]
Had the same issue, this works for me: 

`import {Options} from 'sortablejs';`
..
```
eventOptions: Options = {
    onUpdate: () => this.eventUpdateCounter ++
  };
```

**Comment by scuglik** [comment]
I needed options for dragging beetween multiple list. I marked options as any type in typecsript without any imports and it works.... But only as temporary solution I hope import will be fixed soon :)

`options: any = {
        group: 'groups',
        draggable: '.draggable',
 };`

**Comment by tehZeno** [comment]
> I needed options for dragging beetween multiple list. I marked options as any type in typecsript without any imports and it works.... But only as temporary solution I hope import will be fixed soon :)
> 
> `options: any = { group: 'groups', draggable: '.draggable', };`

did you try my suggestion? (`import {Options} from 'sortablejs';`)

(strangely enough I only get onStart events, but for other options should hopefully just work)

**Comment by scuglik** [comment]
> did you try my suggestion? (`import {Options} from 'sortablejs';`)

Yes with a little edit `import { SortableOptions } from 'sortablejs'; ` to keep same types as in example and it works. Thank you very much
